contract/r/gnoswap/v1/gov/governance/proposal.gno

```diff
@@ -220,6 +220,10 @@ func (p *Proposal) ConfigVersion() int64 {
 	return p.configVersion
 }
 
+func (p *Proposal) IsProposer(addr std.Address) bool {
+	return p.proposer == addr
+}
```

Comment:
This function is only called from the `assertCallerIsProposer` function. it's a simple conditional statement, I think it would be good to inline it.